app/helpers/validators_helper.rb

```diff
@@ -43,7 +42,12 @@ def current_software_version(batch, network)
   def skipped_vote_percent(validator, batch, skipped_vote_percent_best)
     vahl = validator.vote_account_last&.vote_account_history_for(batch.uuid)
     return unless vahl
+
     skipped_votes_percent = (vahl.slot_index_current.to_i - vahl.credits_current.to_i)/vahl.slot_index_current.to_f
     ((skipped_vote_percent_best - skipped_votes_percent.to_f)*100.0).round(2)
   end
+
+  def above_33percent_concentration?(validator)
```

Comment:
Wouldn't it be better to have it in the model?

Reply:
It's related to the view, so I don't think so.